.github/workflows/devbuild.yml

```diff
@@ -57,6 +57,7 @@ jobs:
       run: |
         sudo pip3 install setuptools
         sudo pip3 install numpy pytest flake8
+        sudo pip3 install pyside6 shiboken6
```

Comment:
Although I install pyside6 and shiboken6 on ubuntu, cmake in modmesh does not build with them.  So `-DMODMESH_USE_PYSIDE` is not enabled on ubuntu CI yet.